ale/base/type_sensor.py

```diff
@@ -24,7 +24,7 @@ def line_scan_rate(self):
           2d list of scan rates in the form: [[start_line, line_time, exposure_duration], ...]
         """
         t0_ephemeris = self.ephemeris_start_time - self.center_ephemeris_time
-        return [[float(self.starting_detector_line)], [t0_ephemeris], [self._line_exposure_duration]]
+        return [[0.5], [t0_ephemeris], [self.exposure_duration]]
```

Comment:
Why is the starting detector line hardcoded to 0.5? doesn't it change based on the instrument?

Reply:
This shouldn't have been the starting detector line, this should be the ISIS starting **image** line. Hence it's 0.5